src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -54,6 +55,71 @@ public void setUp() {
     writerVisitor = new ImportWriterVisitor(CURRENT_PACKAGE);
   }
 
+  @Test
```

Comment:
Add a test for something like `new ArrayList<>()`?

Reply:
sure, added a basic unit test for it.